### DISCUSSION TITLE
Dealing with subset situation on disjunction

### DISCUSSION THREAD

**Discussion by conikeec** [cue-lang/cue#1247] [Ideas]
Refer https://github.com/cue-lang/cue/issues/1242

cc @verdverm

**Comment by verdverm** [comment]
What is the focus of this referral here? Could you add more to the description to direct and contextualize this discussion?

**Comment by conikeec** [comment]
I thought you had suggested we start a discussion based on the comment here https://github.com/cue-lang/cue/issues/1242#issuecomment-910816528 .. Hence I did ..

The goal is to 
1. Get some additional thoughts of yours as you suggest
2. I am building some very complex models in a certain domain and would be great to collaborate and seek your ideas (if ok by you)

LMK if I am in line with your expectations

**Reply by verdverm** [reply]
- The idea for a discussion is because it is a better place to capture the solution to a common problem, at least temporarily. I think long term they should end up in the docs. 
- My questions here were to also have the leading discussion text contain sufficient context so that others don't have to click through the link and figure it out. You should be able to edit the description to add that.
- My collab comment was more about documentation for the example we worked through as well. I have some ideas for adding this to a new section "common confusions" for https://cuetorials.com
- There is also ample need for discussing data modelling practices, tips, and tricks; which I am also happy to participate in

**Comment by mpvl** [comment]
I think detecting subsumed and thus redundant values in disjunctions would indeed be a nice job for `vet`.
